src/index.js

```diff
@@ -144,26 +447,39 @@ export default class OsmRequest {
   }
 
   /**
-   * Add or replace a property in a given element
+   * Find an element with it OsmId within an OSM collection
+   * @param {Object} json An object with key that can be 'node', 'way', 'relation'
+   * @param {string} osmId Eg: node/12345
+   * @return {Obejct} OSM element
+   */
+  findElementWithinOSMCollection(json, osmId) {
+    const elementType = findElementType(osmId);
+    const elementId = findElementId(osmId);
+    if (elementType in json) {
+      return json[elementType].find(element => element.$.id === elementId);
+    }
+    return undefined;
+  }
+
+  /**
+   * Add or replace a tag in a given element
    * @param {Object} element
-   * @param {string} propertyName
-   * @param {string} propertyValue
+   * @param {string} tagName
```

Comment:
Those two arguments must use the same naming as in the function. So if you change that, change the funciton name too. But, it would brake the API so...

Reply:
This will be changed in next weeks, in order to have coherent naming. To avoid breaking API, we will have for some time aliases + deprecated tag in doc to alert developers using the library.